src/AdoNetCore.AseClient/Enum/BackupServerSeverity.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace AdoNetCore.AseClient.Enum
```

Comment:
I don't think this is necessary either, since it also relates to parsing the messages.